### PR TITLE
CallToActionCard: updates story from knobs to controls

### LINKED ISSUE #30182: Grafana-ui: update stories to use controls
**What would you like to be added**:
Now that grafana ui storybook is on v6 it'd be great if we migrate away from knobs to controls as detailed [here](https://github.com/grafana/grafana/blob/master/contribute/style-guides/storybook.md#controls).

**Why is this needed**:
Controls infers props through types and auto generates the form controls that knobs creates. This results in less code and so less maintenance. Below is a list of all remaining stories that make use of `addon-knobs`.

- [x] Alert/Alert.story.tsx #31145 
- [x] Badge/Badge.story.tsx https://github.com/grafana/grafana/pull/30180
- [x] BarGauge/BarGauge.story.tsx #31223
- [x] BigValue/BigValue.story.tsx #31240
- [x] Button/Button.story.tsx https://github.com/grafana/grafana/pull/29433
- [x] ButtonCascader/ButtonCascader.story.tsx #31288
- [x] CallToActionCard/CallToActionCard.story.internal.tsx #31393 
- [x] Card/Card.story.internal.tsx (This doesn't have any knobs)
- [x] Cascader/Cascader.story.tsx #31399
- [x] ClipboardButton/ClipboardButton.story.internal.tsx #31422
- [x] ColorPicker/ColorPicker.story.tsx #31429
- [x] ColorPicker/NamedColorsPalette.story.tsx #31443
- [x] ConfirmButton/ConfirmButton.story.tsx #31476
- [x] ConfirmModal/ConfirmModal.story.tsx #31781
- [x] Drawer/Drawer.story.tsx #31821
- [x]  Dropdown/ButtonSelect.story.internal.tsx 
- [x] FormField/FormField.story.internal.tsx #32018
- [x] Forms/FieldValidationMessage.story.tsx #32020
- [x] Forms/Form.story.tsx #32022
- [x] Forms/Legacy/Input/Input.story.internal.tsx #32059
- [x] Forms/Legacy/Select/Select.story.internal.tsx #32225
- [x] Forms/Legacy/Switch/Switch.story.internal.tsx #32249
- [x] Forms/Legend.story.tsx #32250
- [x] Forms/RadioButtonGroup/RadioButtonGroup.story.tsx #32251
- [x] Graph/Graph.story.tsx #32264
- [x] Graph/GraphLegend.story.tsx (not found in the codebase)
- [x] Graph/GraphWithLegend.story.tsx #32285
- [x] GraphNG/GraphNG.story.tsx #32294
- [x] InfoBox/InfoBox.story.tsx
- [x] Input/Input.story.tsx #32712
- [x] Layout/Layout.story.tsx #32761
- [x] List/List.story.internal.tsx #32832
- [x] Modal/Modal.story.tsx #32840
- [x] Pagination/Pagination.story.tsx #33045
- [x] PieChart/PieChart.story.tsx
- [x] SecretFormField/SecretFormField.story.internal.tsx #33920
- [x] Select/Select.story.tsx #33954
- [x] Slider/RangeSlider.story.tsx #33980
- [x] Slider/Slider.story.tsx #33983
- [x] Spinner/Spinner.story.tsx #33988
- [x] StatsPicker/StatsPicker.story.tsx #34008
- [x] Switch/Switch.story.tsx #34016
- [x] Table/Table.story.tsx #34193
- [x] TextArea/TextArea.story.tsx #34195
- [x] Tooltip/Tooltip.story.tsx #33130
- [x] ValuePicker/ValuePicker.story.tsx #33122
- [x] VizLayout/VizLayout.story.tsx #33043
- [x] VizLegend/VizLegend.story.tsx #32801

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR updates story from knobs to controls
**Which issue(s) this PR fixes**:
Relates to  #30182

### CODE REVIEW

File: packages/grafana-ui/src/components/CallToActionCard/CallToActionCard.story.internal.tsx [hunk 1]
```diff
@@ -1,36 +1,54 @@
...
 };
 
-export const basic = () => {
+interface StoryProps extends Partial<CallToActionCardProps> {
+  Element: string;
+  H1Text: string;
+  buttonText: string;
+}
+
+export const basic: Story<StoryProps> = (args) => {
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/CallToActionCard/CallToActionCard.story.internal.tsx:31]:
```suggestion
export const Basic: Story<StoryProps> = (args) => {
```
The stories are basically React components, so should be capitalised. I know we have a lot of lowercased story names in the code, but we're trying to fix those.

**Review by Clarity-89 — APPROVED** [review]
Looks good!

File: packages/grafana-ui/src/components/CallToActionCard/CallToActionCard.story.internal.tsx (hunk 1 above)
Comment (thisisobate) [reply packages/grafana-ui/src/components/CallToActionCard/CallToActionCard.story.internal.tsx:31]: Alright. Thanks! @Clarity-89

**Review by hugohaggmark — APPROVED** [review]
LGTM 🎉 !

**Pull request merged**